.github/workflows/continuous-delivery.yml

```diff
@@ -487,6 +498,71 @@ jobs:
           # TAGS could be separated by newlines or commas
           image=$(sed -n '1{s/,.*//; p}' <<< "$TAGS")
           echo "::set-output name=image::${image}"
+      -
+        name: Generate manifest for operator deployment
+        id: generate-manifest
+        run: |
+          make generate-manifest
+      -
+        name: Upload the operator manifest as artifact in workflow
+        if: always()
```

Comment:
Why do we use `if: always()` here? What step could fail before this without compromising the manifest generation?

Reply:
If publishing the manifest in the artifacts repo in a separate Job, we will need to download the manifest as an artifact before publishing it since different jobs don't share files directly.